app/src/pages/outside/loginPage/pageBlocks/forgotPasswordBlock/forgotPasswordForm/forgotPasswordConfirmation.scss

```
@@ -0,0 +1,53 @@
+  @media (max-width: $SCREEN_XS_MAX) {
+    max-width: 280px;
+  }
+}
+
+.email-value {
+  margin: 0 0 48px 0;
+  color: var(--rp-ui-base-almost-black);
+  font-weight: var(--rp-ui-base-fw-medium);
+  font-size: 13px;
+  line-height: 20px;
+  word-break: break-word;
```

Stylelint (17.13.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)